packages/cli/src/commands/ci/run.ts

```diff
@@ -17,7 +17,8 @@ export default class CiRun extends Command {
   ]
 
   static flags = {
-    app: flags.string({char: 'a', description: 'app name'}),
+    app: flags.string(),
```

Comment:
Why remove the short code and description here? Should this flag stay a string?

Reply:
Oops, nice catch. I meant to change this to `flags.app`. When it's `flags.string`, the `remote` flag doesn't work. And when it's `flags.app`, there's a default short code and description.